internal/docker/podman.go

```diff
@@ -0,0 +1,273 @@
+package docker
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/anchore/stereoscope/internal/log"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/homedir"
+	"github.com/pkg/errors"
+	"golang.org/x/crypto/ssh"
+)
+
+var (
+	// Podman supports the following connections
+	// rootless "unix://run/user/$UID/podman/podman.sock" (Default)
+	// rootfull "unix://run/podman/podman.sock (Default)
+	// remote rootless ssh://engineering.lab.company.com/run/user/$UID/podman/podman.sock
+	// remote rootfull ssh://root@10.10.1.136:22/run/podman/podman.sock
+
+	localRootlessPathTemplate = "/run/user/%d/podman/podman.sock"
+	defaultRemoteRootless     = "ssh://core@localhost:63753%s?secure=false"
```

Comment:
is `63753` a port number that is not expected to change from system to system? Is it driven by podman configuration at all?

Reply:
In Podman's docs and code I don't see info either it changes or not. Podman does offer a config file where we can get this info from. Vide comment below

Reply:
resolved via config file